Automattic/wp-calypso
# Customer Home: Delete unused/redundant code related to WordPress app in Customer Home

A `jetpack/app-branding` feature flag was added in https://github.com/Automattic/wp-calypso/pull/6681 in order to conditionally replace mentions of the WordPress app with the Jetpack app. As the flag is now set to `true` across all environments, we can safely remove all code that mentions the WordPress app, as it is now redundant.

This PR is part of a series that will eventually allow us to remove the `jetpack/app-branding` feature flag.

## Proposed Changes

* This PR iterates on recent changes to mentions of the mobile app in Customer Home. The now redundant code associated with the `jetpack/app-branding` conditional is removed, along with any connected styles or functionality.  

## Testing Instructions

Navigate to Customer Home and verify that there are no changes to style or functionality of the following places where the Jetpack app is promoted:

* The mobile card in the right sidebar on desktop devices. This same card is also styled slightly differently at the bottom of the page on mobile browsers.
* The app setup task in the task carousel at the top of the page.

### Setup Task

<img width="1062" alt="Screenshot 2023-03-10 at 09 59 56" src="https://user-images.githubusercontent.com/2998162/224286178-0bc62966-d025-4489-9da8-5ec42e34184d.png">

### Customer Home Card 

| Desktop  | Mobile |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/224299663-982ab837-a4de-43a6-bbba-b9f6e76df62e.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/224299827-ffcdae96-faa4-411a-bf9f-69530b7a9727.png" width="100%"> |

## Pre-merge Checklist
